magento/magento2
# magento/magento2#32679: setup:upgrade --dry-run does not work as expected by other dry-run-commands in Linux-environment

- Switched --dry-run-Parameter to VALUE_NONE in UpgradeCommand.php
- Switched --dry-run-Parameter to VALUE_NONE in InstallCommand.php

### Description (*)

setup:upgrade and setup:install does not require a value of true when using the --dry-run-parameter. This is now the same beaviour for a dry run as in other Linux-Tools (eg. rsync). As before this fix using --dry-run did not start a dry run for the command, the usage of --dry-run=true was required.

### Fixed Issues (if relevant)
1. Fixes magento/magento2#32679

### Manual testing scenarios (*)
1. Change a module version
2. Run setup:upgrade --dry-run
3. The module version should not be upgraded in the setup_module-table
